bedrock/mozorg/templates/mozorg/about/forums/forums.html

```diff
@@ -73,13 +76,23 @@
         <li id="{{ forum.dashed }}">
           <div class="summary">
             <h4>{{ forum.id }}<a href="#{{ forum.dashed }}">#</a></h4>
-            <p class="description">{{ forum.description|safe }}</p>
+            <p class="description">
+              {%- if forum.id in mdn_forums -%}
```

Comment:
Ah, I see what you mean about the hard coded links below, yes that probably makes sense. Hmm, I'm not sure what's best here. This works does work...